pytket/pytket/tailoring/__init__.py

```diff
@@ -13,9 +13,8 @@
 # limitations under the License.
 
 """
-The circuit module provides an API to interact with the tket
-:py:class:`Characterisation` suite, with methods for noise shaping, characterisation and
-mitigation. This module is provided in binary form during the PyPI installation.
+The `tailoring` module provides an API to interact with tket's charcterisation suite,
```

Comment:
Maybe this isn't the PR for it, but faithfully the `tailoring` module only provides an API for using frame randomisation methods, and maybe we should rename the whole module to frame randomisation in another PR?

If we save that for another review, it should be `tket's characterisation suite`.

Reply:
Done. (Just me being lazy, I saw it was referring to the wrong module name and thought I'd correct it at the same time. But yes it should be a new PR.)